src/assembler.jl

```diff
@@ -192,11 +192,15 @@ end
     permutation = A.permutation
     sorteddofs = A.sorteddofs
     @boundscheck checkbounds(K, dofs, dofs)
+    
+    ## This code is not okay on the GPU (i.e. has to be static code)##
     resize!(permutation, ld)
     resize!(sorteddofs, ld)
     copyto!(sorteddofs, dofs)
-    sortperm2!(sorteddofs, permutation)
-
+    # sort permutation according to sorteddofs (e.g. sorteddofs = [2,5,3,6] -> permutation = [1,3,2,4])
+    sortperm2!(sorteddofs, permutation) 
```

Comment:
```suggestion
    sortperm2!(sorteddofs, permutation)
```
CI is failing due to these trailing whitespaces btw